std/node/os.ts

```diff
@@ -89,6 +89,17 @@ interface UserInfo {
   homedir: string;
 }
 
+arch[Symbol.toPrimitive] = (): string => arch();
```

Comment:
Why not using `arch[Symbol.toPrimitive] = arch;` directly ?
Same applie for the rest of the symbols.

Reply:
Because that's how Node.js does it. I figured that if we're adding a compatibility shim, we might as well aim for 100% fidelity. :-)

There's an observable difference in that `os.arch[Symbol.toPrimitive].name` is the empty string in Node.js but it would be `'arch'` with your suggestion, i.e., it'd print differently.

Reply:
If there is a unit test for this use case on node side, it would be mandatory to do as node do.
Can you check ?

Reply:
I know the answer but it's the wrong question to ask, IMO. Node is so widely used that chances are > 50% that there's some code somewhere depends on this particular quirk. Even if Node's test suite didn't test for it, that doesn't mean we can just deviate.

Random example: people complained their code broke when the `util.inspect()` representation of streams changed in a really minor way. When the user base is big enough, _any_ change is going to get noticed by someone.